src/apps/tco2_dashboard/app.py

```diff
@@ -1956,6 +1984,92 @@ def download_carbonmarket(n_clicks):
     return dict(content=fig.to_html(), filename=filename)
 
 
+@app.callback(
+    Output("download_csv_BCT", "data"),
+    Input("download_btn_BCT", "n_clicks"),
+    prevent_initial_call=True,
+)
+def download_csv_BCT_callback(n_clicks):
+    df = cache.get("bct_carbon")
+    filename = "BCT.csv"
+    return dcc.send_data_frame(df.to_csv, filename=filename)
```

Comment:
we should pass `index=False` here so that the CSVs don't have a meaningless index column

Reply:
@originalpkbims